{
  "Header": {
    "Original": "CVE-2017-14136 (OpenCV (Open Source Computer Vision Library) 3.3 has an out-of-bounds  ...)",
    "Line": 349016,
    "ID": "CVE-2017-14136",
    "Description": "(OpenCV (Open Source Computer Vision Library) 3.3 has an out-of-bounds  ...)"
  },
  "Annotations": [
    {
      "Original": "- opencv \u003cnot-affected\u003e (Incomplete patch never shipped)",
      "Line": 349017,
      "Type": "package",
      "Package": "opencv",
      "Kind": "not-affected",
      "Description": "Incomplete patch never shipped"
    },
    {
      "Original": "NOTE: https://github.com/opencv/opencv/issues/9443",
      "Line": 349018,
      "Type": "NOTE",
      "Description": "https://github.com/opencv/opencv/issues/9443"
    },
    {
      "Original": "NOTE: https://github.com/opencv/opencv/pull/9448",
      "Line": 349019,
      "Type": "NOTE",
      "Description": "https://github.com/opencv/opencv/pull/9448"
    }
  ]
}